app/indexers/hyrax/linked_data_attributes_indexer.rb

```diff
@@ -0,0 +1,66 @@
+module Hyrax
+  # Indexes labels for linked_data_attributes into solr.
+  # @example Add linked_data_attributes to be indexed into `self.linked_data_attributes` as symbols.
+  #   self.linked_data_attributes = [:based_near]
+  #
+  # @todo This replicates current Hyrax DeepIndexingService behaviour of fetching from URI whenever
```

Comment:
Is there still a todo item here?

Reply:
Yes. I'm still retrieving from URI each time to_solr is called ... I'd like to change that. But it's an enhancement so I thought I'd get this finished first.